### ISSUE TITLE
Update download fails

### ISSUE THREAD

**Issue by agentxan** [Prowlarr/Prowlarr#1655]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

I get an error when I try to update prowlarr. I have slow internet and the download fails after 100 seconds. If I download using curl it completes fine. If it helps I was having the same problem with sonarr and radarr and it was fixed in these commits.

https://github.com/Sonarr/Sonarr/commit/63654b99f22b87b42acd699ac5b453f2de20211f
https://github.com/Radarr/Radarr/commit/2042ffce62d8c9e9c0e1e30119138b4d1d22e00a

### Expected Behavior

The download to complete  and run the update.

### Steps To Reproduce

In the update tab click install. After 100 seconds it gives a failed message.

### Environment

```markdown
- OS: Windows 10 22H2
- Prowlarr: v1.4.1.3258
- Docker Install: no
- Using Reverse Proxy: no
- Browser: Firefox 112.02
```

### What branch are you running?

Nightly

### Trace Logs?

[prowlarr.trace.txt](https://github.com/Prowlarr/Prowlarr/files/11447670/prowlarr.trace.txt)

### Trace Logs have been provided as applicable. Reports may be closed if the required logs are not provided.

- [X] I have followed the steps in the wiki link above and provided the required trace logs that are relevant and show this issue.

**Comment by agentxan** [comment]
Yes both radar and sonarr complete there update download in 3 to 4 minutes

**Comment by Qstick** [comment]
The only real difference in Prowlarr is that http stack is async, otherwise as @mynameisbogdan said, the fix you reference is already there.

**Comment by agentxan** [comment]
so what would cause the download to stop at 100 seconds even though data is still being transferred. Should it not only time out if no data is being transferred. If not then as the update file gets bigger and bigger then faster and faster internet connections will be needed to be able to get the update in the 100 seconds.

**Comment by ilike2burnthing** [comment]
@agentxan previously raised the same issue with Jackett - https://github.com/Jackett/Jackett/issues/12711.

Timeout was extended to 300s just for updates - https://github.com/Jackett/Jackett/commit/4a6d630076abeeb2fce1f0d6795a3548a8a3499a.
